src/app/onion/learning-object-builder/pages/materials-page/materials-page.component.html

```diff
@@ -1,6 +1,6 @@
 <clark-column-wrapper [columns]="'lm'">
-  <div left>
-    <clark-scaffold></clark-scaffold>
+  <div left *ngIf="learningObject$">
+    <clark-scaffold [learningObject]= "learningObject$"></clark-scaffold>
```

Comment:
Is this an observable here? I don't believe the input will be compatible with an observable.

Reply:
Resolved